stories/topics/proc-gcp-generate-restore-yml-file.adoc

```diff
@@ -16,15 +16,15 @@ Providing the following output:
 ----
 ===============================================
 Playbook: gcp_restore_deployment
-Description: This playbook is used to restore the AoC Self-managed GCP environment.
+Description: This playbook is used to restore the Ansible Automation Platform from GCP Marketplace environment from a backup.
 -----------------------------------------------
-This playbook is used to restore the AoC Self-managed GCP environment.
+This playbook is used to restore the Ansible Automation Platform from GCP Marketplace environment from a backup.
 For more information regarding backup and restore, visit our official documentation - 
-
+https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/red_hat_ansible_automation_platform_from_gcp_marketplace_guide/assembly-gcp-backup-and-restore
```

Comment:
For more information on backup and restore, see link:https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/red_hat_ansible_automation_platform_from_gcp_marketplace_guide/assembly-gcp-backup-and-restore[TITLE] as above